Singleton/SceneInstance.cs

```diff
@@ -2,16 +2,16 @@
 
 public class SceneInstance<T> : MonoBehaviour where T : class, new() {
 	//
-	private static T _Instance = null;
+	protected static T _Instance = null;
```

Comment:
it's used by the game, this is a shared library we use in the game as well to define singletons
i also don't see the issue here, let's chat in person about this